app/code/community/Razorpay/Payments/Model/Paymentmethod.php

```diff
@@ -174,6 +174,26 @@ public function _getChannel()
         return sprintf(self::CHANNEL_NAME, $edition, Mage::getVersion(), self::VERSION);
     }
 
+    protected function hash_equals($str1, $str2)
```

Comment:
http://stackoverflow.com/questions/32671908/hash-equals-alternative-for-php-5-5-9